### PR TITLE
fixed kafka consumer group issue

### PR DESCRIPTION
**Pull request by aman-tunerlabs**

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -59,6 +59,7 @@ DISABLE_LEARNER_SERVICE_ON_OFF = ""
 KAFKA_COMMUNICATIONS_ON_OFF = ""
 KAFKA_URL = "" // IP address of kafka server with port without HTTP
 NOTIFICATIONS_TOPIC = "OFF/TOPIC_NAME"
+KAFKA_GROUP_ID = ""
```
Comment (aks30) [inline .env.sample:62]: @aman-tunerlabs - Can you add it envVariablesHelper so that it is mandatory value?

File: config/kafka-config.js [hunk 2]
```diff
@@ -66,20 +86,18 @@ var connect = function (config) {
   * @param {Boolean} [commit = false] - kafka commit. By default set to false.
 */
 
-var _sendToKafkaConsumers = function (topic,client, commit = false) {
+var _sendToKafkaConsumers = function (topic,host, commit = false) {
```
Comment (aks30) [inline config/kafka-config.js]: @aman-tunerlabs - Please change commit to commitOffset variable name

File: config/kafka-config.js [hunk 3]
```diff
@@ -44,10 +45,29 @@ var connect = function (config) {
...
+  _sendToKafkaConsumers(
+    config.topics["languagesTopic"],
+    config.host,
+    true
+  );
+
+  _sendToKafkaConsumers(
+    config.topics["emailTopic"],
+    config.host, 
+    false
```
Comment (aks30) [inline config/kafka-config.js]: @aman-tunerlabs - Let us remove that commit setting, let it be commonly true for all.

File: .env.sample (hunk 1 above)
Comment (aman-tunerlabs) [reply .env.sample:62]: @aks30  envVariables is not there in kendra as of now.

**Pull request merged**